admin/spam/views.py

```diff
@@ -107,32 +106,31 @@ def get_context_data(self, **kwargs):
         return kwargs
 
     def form_valid(self, form):
-        spam_id = self.kwargs.get('spam_id')
```

Comment:
Why did you stop caching the `spam_id`?

Reply:
I just thought this looked cleaner.

Reply:
Really? With all the `{self.kwargs.get("spam_id")}`s sprinkled throughout the f-strings, rather than a nice `{spam_id}`?